CONTRIBUTING.md

```diff
@@ -131,17 +131,16 @@ pub global run flutter_plugin_tools xctest --target RunnerUITests --skip <plugin
 We gladly accept contributions via GitHub pull requests.
 
 Please peruse our
-[style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) and
-[design principles](https://flutter.io/design-principles/) before
```

Comment:
This design principles link now redirects to https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo